<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusSine</class>
 <widget class="QDialog" name="MotusSine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="hengyaolabel">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横摇</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangvalueEdit">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>188</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyivalueEdit">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>188</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingvalueEdit">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>188</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="shengjianglabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升降</string>
   </property>
  </widget>
  <widget class="QLabel" name="poslabel">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>幅值°/mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="zongqinglabel">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵倾</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongvalueEdit">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>188</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangvalueEdit">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>188</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="speedlabel">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>220</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>频率Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="hangxianglabel">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航向</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongfreEdit">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>218</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaofreEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>218</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingfreEdit">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>218</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaovalueEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>188</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyifreEdit">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>218</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="qianchonglabel">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>前冲</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangfreEdit">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>218</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangfreEdit">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>218</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hengyilabel">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sinStartButton">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>正弦开始</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyiphaseEdit">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>249</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangphaseEdit">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>249</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangphaseEdit">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>249</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongphaseEdit">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>249</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaophaseEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>249</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingphaseEdit">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>249</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phaselabel">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>相位0-360</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buffStopButton">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>缓冲停止</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueEdit">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>344</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="accEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>344</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="freEdit">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>344</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="speedEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>344</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueedit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>399</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="freedit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>399</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="speededit">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>399</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="accedit">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>399</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calButton">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>338</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calbutton">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>391</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>324</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>幅值°/m</string>
   </property>
  </widget>
  <widget class="QLabel" name="FreLabel">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>322</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>频率Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>324</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>速度°m/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="accLabel">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>324</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>加速度°m/s²</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>378</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>频率Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedlabel_3">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>378</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>速度°m/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="acclabel">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>378</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>加速度°m/s²</string>
   </property>
  </widget>
  <widget class="QLabel" name="valuelabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>幅值°/m</string>
   </property>
  </widget>
  <widget class="QLabel" name="messagelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>注意：平台在中位时进行正弦运动</string>
   </property>
  </widget>
  <widget class="QLabel" name="qianchonglabel_2">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>前冲</string>
   </property>
  </widget>
  <widget class="QLabel" name="zongqinglabel_2">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵倾</string>
   </property>
  </widget>
  <widget class="QLabel" name="shengjianglabel_2">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升降</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangspeedEdit">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>60</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hangxianglabel_2">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航向</string>
   </property>
  </widget>
  <widget class="QLabel" name="posremarklabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>120</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>位置：纵倾、横摇、航向单位为度，横移、前冲、升降单位为毫秒</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongposEdit">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>30</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="speedlabel_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedremarklabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>140</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>速度：纵倾、横摇、航向单位为度/秒，横移、前冲、升降单位为毫米/秒</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingspeedEdit">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>60</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangspeedEdit">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>60</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hengyilabel_2">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横移</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangposEdit">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>30</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyispeedEdit">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>60</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="carryOutButton">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongspeedEdit">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>60</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="poslabel_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingposEdit">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>30</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaoposEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyiposEdit">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>30</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hengyaolabel_2">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横摇</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangposEdit">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>30</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaospeedEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
